pkg/extractor/extractor_test.go

```diff
@@ -0,0 +1,102 @@
+package extractor
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type toAbsolutePathsTestCase struct {
+	name string
+	args struct {
+		path string
+	}
+	expected struct {
+		path string
+	}
+}
+
+func test_existed_file() *toAbsolutePathsTestCase {
+	p, _ := filepath.Abs("../../internal/fixtures/kube/pass-all.yaml")
+	return &toAbsolutePathsTestCase{
+		name: "existed file, should return abs path with no errors",
+		args: struct{ path string }{
+			path: "../../internal/fixtures/kube/pass-all.yaml",
+		},
+		expected: struct {
+			path string
+		}{
+			path: p,
+		},
+	}
+}
+
+func test_not_existed_file() *toAbsolutePathsTestCase {
+	return &toAbsolutePathsTestCase{
+		name: "test not existed file, should return an error",
+		args: struct{ path string }{
+			path: "../../internal/fixtures/kube/bla.yaml",
+		},
+		expected: struct {
+			path string
+		}{
+			path: "",
+		},
+	}
+}
+
+func test_directory_file() *toAbsolutePathsTestCase {
+	return &toAbsolutePathsTestCase{
+		args: struct{ path string }{
+			path: "../../internal/fixtures/kube",
+		},
+		expected: struct {
+			path string
+		}{
+			path: "",
+		},
+	}
+}
+
+func TestToAbsolutePath(t *testing.T) {
+	tests := []*toAbsolutePathsTestCase{
+		test_existed_file(),
+		test_directory_file(),
+		test_not_existed_file(),
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			absolutePath, _ := ToAbsolutePath(tt.args.path)
+			assert.Equal(t, tt.expected.path, absolutePath)
+		})
+	}
+}
+
+func TestExtractConfigurationsFromYamlFile(t *testing.T) {
+	t.Run("invalid yaml path, should return invalid file with returned ToAbsolutePath error", func(t *testing.T) {
+		actualVal, actualStr, actualErr := ExtractConfigurationsFromYamlFile("")
```

Comment:
I think that `actualVal, actualStr, actualErr` are not easy to understand what we expect the function to return, if I want to know it I need to go the function declaration. What do you think about renaming in to something more meaningful like `configurations, absolutePath, ...`